### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Johndowne** [mbuchber/ha_heliotherm#17] [General]
Would anyone be able to clarify the meanings of the various kWh figures? I am particularly interested in the heating (not hot water) electricity input and heating energy output.
I have a Heliotherm air source heat pump.

**Comment by Johndowne** [comment]
As far as I can tell …
stromzahler_gesamt gives me the cumulative electrical energy consumed and 
wmz_heizung Is the cumulative heat energy output by the heat pump.

Can anyone confirm this is the case?

Thanks

**Reply by kobius77** [reply]
that's correct (but you probably figured it out yet 8))

**Comment by Johndowne** [comment]
Thanks yes. It is a shame that only whole numbers are recorded meaning that in warmer winter weather the daily CoP can vary widely. I also concluded that sensor.heliotherm_heatpump_hkr_soll_temperatur is the “weather compensated” / heat curve determined return (I think) temperature the heat pump seeks to achieve so that as sensor.heliotherm_heatpump_temp_aussen (outside weather temp) decreases, the return temp increases according to the heat curve defined figures.

**Comment by kobius77** [comment]
> Thanks yes. It is a shame that only whole numbers are recorded meaning that in warmer winter weather the daily CoP can vary widely.

i just found this integration and installed today - heatpump didn't run since :(

but there's an issue open, and looking at the history of the repo it seems @mbuchber has been working on the data type used - which might (have) solve(d) these issues (yours and the one posted).

> I also concluded that sensor.heliotherm_heatpump_hkr_soll_temperatur is the “weather compensated” / heat curve determined return (I think) temperature the heat pump seeks to achieve

correct.
